sdk/typescript/src/signers/txn-data-serializers/type-tag-serializer.ts

```diff
@@ -3,7 +3,7 @@
 
 import { splitGenericParameters } from '@mysten/bcs';
 import type { TypeTag } from '../../types/index.js';
-import { normalizeSuiAddress } from '../../types/index.js';
+import { normalizeSuiAddress } from '../../utils/sui-types.js';
```

Comment:
this helpers were imported from types/index a lot, meaning that we are importing all of the types all over the place, this change enables us to use type-only imports of all the types for the client